docs/scos/dev/glue-api-guides/202204.0/glue-backend-api/security-and-authentication.md

```diff
@@ -0,0 +1,29 @@
+---
```

Comment:
we have almost the same document here docs\scos\dev\glue-api-guides\202204.0\security-and-authentication.md. Which one should be removed? Or you want to let both live for some reason?

Reply:
Seems like `docs\scos\dev\glue-api-guides\202204.0\security-and-authentication.md` is outdated.

@gechetspr what do you think?

Reply:
We need them both for now. They are similar, but for the current and deprecated approach. We will remove old one, when the new infrastructure will be widely used